splash/render_options.py

```diff
@@ -248,11 +251,23 @@ def get_common_params(self, js_profiles_path):
             # 'lua': self.get_lua(),
         }
 
+    def get_image_params(self):
+        return {
+            'width': self.get_width(),
+            'height': self.get_height(),
+            'scale_method': self.get_scale_method()
+        }
+
     def get_png_params(self):
-        return {'width': self.get_width(), 'height': self.get_height(),
-                'scale_method': self.get_scale_method()}
+        return self.get_image_params()
+
+    def get_jpeg_params(self):
+        params = {'quality': self.get_quality()}
```

Comment:
Ah, ok. I skimmed through the sources and remembered that both png and jpeg functions had some argument to apply (for png it is complevel), and it felt somehow "unfair" to only have one of those functions to have this parameter overridable.